ArduPlane/Parameters.cpp

```diff
@@ -1317,6 +1317,8 @@ const AP_Param::ConversionInfo conversion_table[] = {
     { Parameters::k_param_arming,             3,      AP_PARAM_INT8,  "ARMING_RUDDER" },
     { Parameters::k_param_compass_enabled_deprecated,       0,      AP_PARAM_INT8, "COMPASS_ENABLE" },
     { Parameters::k_param_arming,           128,     AP_PARAM_INT16,  "ARMING_CHECK" },
+
+    { Parameters::k_param_mission,           1,     AP_PARAM_INT8,  "MIS_RESTART" },
```

Comment:
why aren't we using AP_Param::convert_parameter_width() ?

Reply:
removed param change for MAV_CMD_DO_SET_RESUME_REPEAT_DIST waypoint